src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * 40;
+
+  if (days >= 3 && days < 7) {
+    total = total - 20;
+
+    return total;
+  } else if (days >= 7) {
+    total = total - 50;
+  }
```

Comment:
```suggestion
  if (days >= 3 && days < 7) {
    return total - 20;
  }
  
  if (days >= 7) {
    return total - 50;
  }
```
Do you really need to modify total? Just make it a const and return value with discount.
Also, there is no need for `else if` as your code will not go further after `return` anyway.
P.S. would be great to create variables for `20` and `50`.